infrastructure/terraform/modules/activation/variables.tf

```diff
@@ -34,3 +34,13 @@ variable "trigger_function_location" {
   type        = string
   default     = "us-central1"
```

Comment:
Unrelated to this PR, but there is a global region variable. I would remove "default" from here - will help catch misconfiguration.